llpc/context/llpcCompiler.cpp

```diff
@@ -42,7 +42,6 @@
 #include "llpcSpirvLower.h"
 #include "llpcSpirvLowerResourceCollect.h"
 #include "llpcSpirvLowerTranslator.h"
-#include "llpcSpirvLowerUtil.h"
```

Comment:
could  you add this header back? there is dependence on this header in our internal branch

Reply:
done